src/plugins/meter-circular/_base.scss

```diff
@@ -0,0 +1,463 @@
+/*
+ * Web Experience Toolkit (WET) / Boîte à outils de l'expérience Web (BOEW)
+ * wet-boew.github.io/wet-boew/License-en.html / wet-boew.github.io/wet-boew/Licence-fr.html
+ */
+
+.mtr-rnd {
+	background: none;
+	box-shadow: none;
+	height: 9em;
+	line-height: 9em;
+	position: relative;
+	width: 9em;
+
+	&:after {
+		border: .75em solid #f5f5f5;
+		border-radius: 50%;
+		content: "";
+		height: 100%;
+		left: 0;
+		position: absolute;
+		top: 0;
+		width: 100%;
+	}
+
+	> span {
+		height: 100%;
+		overflow: hidden;
+		position: absolute;
+		top: 0;
+		width: 50%;
+		z-index: 1;
+	}
+
+	.mtr-b {
+		background: none;
+		border-style: solid;
+		border-width: .75em;
+		height: 100%;
+		position: absolute;
+		top: 0;
+		width: 100%;
+	}
+
+	.mtr-l {
+		left: 0;
+
+		.mtr-b {
+			-webkit-transform-origin: center left;
+			border-bottom-right-radius: 5em;
+			border-left: 0;
+			border-top-right-radius: 5em;
+			left: 100%;
+			transform-origin: center left;
+		}
+	}
+
+	.mtr-r {
+		right: 0;
+
+		.mtr-b {
+			-webkit-transform-origin: center right;
+			border-bottom-left-radius: 5em;
+			border-right: 0;
+			border-top-left-radius: 5em;
+			left: -100%;
+			transform-origin: center right;
+		}
+	}
+
+	.mtr-val{
+		background: transparent;
+		border-radius: 50%;
+		color: #333;
+		font-size: 2em;
+		height: 100%;
+		text-align: center;
+		width: 100%;
+	}
+
+	&.wlbl {
+		.mtr-val {
+			height: 90%;
+			left: 5%;
+			line-height: 2em;
+			position: absolute;
+			top: 10%;
+			width: 90%;
+		}
+		.mtr-lbl {
+			background: transparent;
+			border-radius: 50%;
+			color: #333;
+			font-size: .75em;
+			height: 90%;
+			left: 5%;
+			line-height: 1em;
+			position: absolute;
+			text-align: center;
+			top: 50%;
+			width: 90%;
+		}
+	}
+
+	/* Color */
+	&.mtr-optimal {
+		.mtr-b {
+			border-color: #236423;
+		}
+	}
+	&.mtr-suboptimal {
```

Comment:
Regarding optimal, sub-optimal and less good are the keyword used to define the meter region in the HTML5 spec. Or do you talk about "mtr" prefix ?